iscsiuio/src/unix/libs/qedi.c

```diff
@@ -58,6 +58,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <sys/sysmacros.h>
+#include <semaphore.h>
```

Comment:
I don't think this is needed, it's for IPC semaphores while the added code here uses a pthread mutex